api/src/main/java/io/strimzi/api/kafka/model/status/KafkaMirrorMakerStatus.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.status;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+/**
+ * Represents a status of the Kafka Mirror Maker resource
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"conditions", "observedGeneration"})
+@EqualsAndHashCode
+public class KafkaMirrorMakerStatus extends Status {
+    private static final long serialVersionUID = 1L;
```

Comment:
could we add information about which are the source and destination clusters in mirroring?

Reply:
How do you think it should look like? Isn't it just mirroring the `spec` section?

Reply:
Yes it is, actually. Maybe it's useless then.